hazelcast-jet-distribution/src/root/changelog.txt

```diff
@@ -1,8 +1,24 @@
 ** Hazelcast Jet 4.2 **
 
-* Generic Debezium-Jet connector and its MySQL & Postgres versions have
-  been wrapped with a proprietary API to make them more convenient to
-  use and more production-ready.
+* We added Change Data Capture (CDC) support for PostgreSQL and MySql. New
+  connectors use Debezium to provide a high-level API for processing a
+  stream changes from relational databases. A new map sink can be used to
+  automatically keep the map in-sync with changes from the database.
+* We added new source and sink connectors for Elasticsearch 5, 6 and 7
```

Comment:
_... with support for slicing and co-located reading for maximum read performance and extended Elasticsearch's client retry mechanism for reliable writing._